docs/specification.md

```diff
@@ -102,10 +102,6 @@ type config =
   // ^ Determine the maximum number of ongoing proposals that are allowed in the contract.
   ; max_votes : nat
   // ^ Determine the maximum number of votes associated with a proposal including positive votes
-  ; max_voting_period : nat
-  // ^ Determine the maximum value of voting period that is allowed to be set.
-  ; min_voting_period : nat
-  // ^ Determine the minimum value of voting period that is allowed to be set.
 
```

Comment:
Oh actually, one last thing: this needs to be updated, as `config` now contains the `voting_period` field as well.

Reply:
Have added `voting_period` and `quorum_threshold` as well, which seemed to be missing as well.

Reply:
Ah great :ok_hand: